R/merge_sce_list.R

```diff
@@ -102,6 +105,17 @@ merge_sce_list <- function(
 
     # Update retain_coldata_cols
     retain_coldata_cols <- c(retain_coldata_cols, altexp_columns)
+
+    # Check each altExp metadata, for SCEs with the altExp
+    names(altexp_attributes) |>
+      purrr::walk(
+        \(altexp_name) {
+          sce_list |>
+            purrr::keep(\(sce) altexp_name %in% altExpNames(sce)) |>
+            purrr::map(altExp, altexp_name) |>
+            check_metadata()
```

Comment:
Yeah, I agree it could sort of go either way. I figure if someone wanted to fully pull out the altexp it might be useful to have this information handy?